kitchen.yml

```diff
@@ -9,11 +9,6 @@ provisioner:
   name: chef_zero
 
 platforms:
-  - name: windows-2012r2
```

Comment:
Sadly 2012r2 is still a thing. https://docs.microsoft.com/en-us/lifecycle/products/windows-server-2012-r2

Reply:
Reverted changes that dropped 2012.